metadata/src/main/java/org/apache/kafka/controller/ClusterControlManager.java

```diff
@@ -244,6 +251,11 @@ boolean check() {
      */
     private final boolean zkMigrationEnabled;
 
+    /**
+     * True if eligible leader replicas is enabled.
+     */
+    private final boolean eligibleLeaderReplicasEnabled;
```

Comment:
I don't see why this is in ClusterControlManager. It's not related to cluster membership -- seems related to replication, so it should go in ReplicationControlManager ?

Reply:
Sure, done.